DataAccess/Repository/SignRepository.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DataAccess.Context;
+using DataAccess.Repository.Interfaces;
+using Domain.EF_Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.Repository
+{
+    public class SignRepository:BaseRepository<Sign>
```

Comment:
you dont need this bacause in dont have any diffrence from BaseRepository, just use IRepositry<Sign> in BL